core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-configuration/components/dot-experiments-configuration-traffic/dot-experiments-configuration-traffic.component.scss

```diff
@@ -7,15 +7,25 @@ h2 {
     text-transform: uppercase;
 }
 
-:host ::ng-deep p-card .p-card-content {
-    .card-load {
-        border-bottom-left-radius: 0;
-        border-bottom-right-radius: 0;
-        border-bottom: 0;
+:host ::ng-deep {
+    dot-icon {
+        color: $gray-light;
+
+        &.isDone {
+            color: $success;
+        }
```

Comment:
Same as before, check the new variables in `libs\dotcms-scss\shared\_colors.scss`